### PR TITLE
Added Keywords to fetch replicasets in namespace

### PR DESCRIPTION
**Pull request by hello2ray**
\<Remember to add meaningful title\>

\<Short description of the PR\>

Fixes #\<issue number\>

Before merge following needs to be applied:
- [ ] At least one example testcase added in testcases/ done
- [ ] Library Documentation regenerated according to [Generate docs](https://github.com/devopsspiral/KubeLibrary#generate-docs) done
- [ ] PR entry added in CHANGELOG.md in **In progress** section done
- [ ] All new testcases tagged as **prerelease** along other tags to exclude it from execution until released on PyPI done
- [ ] Coverage threshold increased in [.coveragerc](https://github.com/devopsspiral/KubeLibrary/blob/master/.coveragerc) if new coverage is higher than actual, see the lint-and-coverage step in CI done
```
fail_under = 82
```

### CODE REVIEW

**Comment by hello2ray** [comment]
@m-wcislo Could you please review now. All checklists are added

**Comment by hello2ray** [comment]
@m-wcislo 
Please generate docs. You have the link in
Library Documentation regenerated according to >>>Generate docs<<<

CHANGELOG.md
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
- Keyword for getiing role and rolebinding [#56](https://github.com/devopsspiral/KubeLibrary/pull/56) by [@satish-nubolab](https://github.com/satish-nubolab)
- Bearer token authentication [#39](https://github.com/devopsspiral/KubeLibrary/pull/39) by [@m-wcislo](https://github.com/m-wcislo)
- Keyoword for create and delete a cronjob [#71](https://github.com/devopsspiral/KubeLibrary/pull/71) by [@satish-nubolab](https://github.com/satish-nubolab)
- Keywords for get replicaset in a namespace [#82](https://github.com/devopsspiral/KubeLibrary/pull/88) by [@hello2ray](https://github.com/hello2ray)
 
@m-wcislo m-wcislo 21 hours ago Collaborator
should be #91

**Comment by hello2ray** [comment]
@review pls 
@Nilsty @m-wcislo @mika-b

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Keyword for getiing role and rolebinding [#56](https://github.com/devopsspiral/KubeLibrary/pull/56) by [@satish-nubolab](https://github.com/satish-nubolab)
 - Bearer token authentication [#39](https://github.com/devopsspiral/KubeLibrary/pull/39) by [@m-wcislo](https://github.com/m-wcislo)
 - Keyoword for create and delete a cronjob [#71](https://github.com/devopsspiral/KubeLibrary/pull/71) by [@satish-nubolab](https://github.com/satish-nubolab)
-## [0.4.0] - 2021-03-12
```
Comment (Nilsty) [inline CHANGELOG.md:14]: Please add the headline for version 0.4.0 back to the CHANGELOG.md

File: src/KubeLibrary/KubeLibrary.py [hunk 2]
```diff
@@ -323,6 +340,14 @@ def filter_deployments_names(self, deployments):
         """
         return [d.metadata.name for d in deployments]
 
+    def filter_replicasets_names(self, replicasets):
+        """Filter deployment  names for list of deployments .
```
Comment (Nilsty) [inline src/KubeLibrary/KubeLibrary.py:344]: Please update the documentation here from `deployments` to `replicasets`
You also need to regenerate the docs then.

File: testcases/replicaset/replicaset_kw.robot [hunk 3]
```diff
@@ -0,0 +1,19 @@
+*** Settings ***
+Library           Collections
+Library           RequestsLibrary
+# For regular execution
+#Library           KubeLibrary
```
Comment (Nilsty) [inline testcases/replicaset/replicaset_kw.robot:5]: Please change the library import to 
```
Library    KubeLibrary
```
This way, this example will work once the keyword is added to the released version.

Comment (Nilsty) [inline testcases/replicaset/replicaset_kw.robot:5]: Please change the library import to 
```
Library    KubeLibrary
```
This way, this example will work once the keyword is added to the released version.

**Review by Nilsty — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (hello2ray) [reply CHANGELOG.md:14]: @Nilsty Done

File: src/KubeLibrary/KubeLibrary.py (hunk 2 above)
Comment (hello2ray) [reply src/KubeLibrary/KubeLibrary.py]: @Nilsty Done

File: testcases/replicaset/replicaset_kw.robot (hunk 3 above)
Comment (hello2ray) [reply testcases/replicaset/replicaset_kw.robot]: @Nilsty Done

**Comment by hello2ray** [comment]
@Nilsty Can u pls validate

File: CHANGELOG.md (hunk 1 above)
Comment (hello2ray) [reply CHANGELOG.md:14]: > Please add the headline for version 0.4.0 back to the CHANGELOG.md

Done

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,7 +11,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Keyword for getiing role and rolebinding [#56](https://github.com/devopsspiral/KubeLibrary/pull/56) by [@satish-nubolab](https://github.com/satish-nubolab)
 - Bearer token authentication [#39](https://github.com/devopsspiral/KubeLibrary/pull/39) by [@m-wcislo](https://github.com/m-wcislo)
 - Keyoword for create and delete a cronjob [#71](https://github.com/devopsspiral/KubeLibrary/pull/71) by [@satish-nubolab](https://github.com/satish-nubolab)
+- Keywords for get replicaset in a namespace [#82](https://github.com/devopsspiral/KubeLibrary/pull/92) by [@hello2ray](https://github.com/hello2ray)
 ## [0.4.0] - 2021-03-12
+=======
```
Comment (m-wcislo) [inline CHANGELOG.md:16]: Please clean this.

**Review by m-wcislo — CHANGES_REQUESTED** [review]

**Comment by hello2ray** [comment]
@m-wcislo @Nilsty  Please review

**Comment by Nilsty** [comment]
It's looking pretty good. 
You still need to push a new version of the index.html as there it's still saying deployments instead of replicasets.
ksnip_20210730-144004

**Comment by hello2ray** [comment]
@Nilsty @m-wcislo Kindly review

**Comment by hello2ray** [comment]
@Nilsty @m-wcislo can we merge this

**Review by m-wcislo — APPROVED** [review]

File: testcases/replicaset/replicaset_kw.robot (hunk 3 above)
Comment (m-wcislo) [reply testcases/replicaset/replicaset_kw.robot]: This is still not done

**Review by m-wcislo — CHANGES_REQUESTED** [review]

**Review by m-wcislo — CHANGES_REQUESTED** [review]
docs still has the issue that @Nilsty pointed out.

**Comment by hello2ray** [comment]
@m-wcislo @Nilsty I created new docs. Kindly check once

**Comment by hello2ray** [comment]
@m-wcislo @Nilsty Could you kindly validate. I have changed the docs pointed by @Nilsty

**Comment by m-wcislo** [comment]
I think you didn't committed the changes then. I still see the old docs in pull request

**Comment by hello2ray** [comment]
@Nilsty @m-wcislo Could you kindly check now

**Review by Nilsty — APPROVED** [review]
LGTM

**Review by m-wcislo — APPROVED** [review]

**Pull request merged**